src/renderer/routes/flow_chart/views/BlockModal.tsx

```diff
@@ -117,6 +120,15 @@ const BlockModal = ({
         <h2 className="text-lg font-semibold text-muted-foreground">
           Python code
         </h2>
+        <Button
+          onClick={async () => {
+            await window.api.openEditorWindow(blockFullPath);
+          }}
+          data-testid="btn-edit-python"
+        >
+          Edit Python Code
+        </Button>
```

Comment:
![image](https://github.com/flojoy-ai/studio/assets/47371088/6b9d5006-1f4c-442d-921f-6e9a49ebf5c4)
this button could afford to be a bit less wide

Reply:
Done.